### DISCUSSION TITLE
to_pandas() failing due to missing sqlglot attribute

### DISCUSSION THREAD

**Discussion by JRocki** [ibis-project/ibis#8153] [Q&A] [answered]
Hello all,

My issue is `to_pandas()` breaks when trying to convert an ibis table to a pandas dataframe. Here's my code:

```
df = pd.DataFrame(
    [["a", 1, 2], ["b", 3, 4]],
    columns=["one", "two", "three"],
    index=[5, 6],
)

t = ibis.memtable(df, name="t")

t.to_pandas()
```
This breaks, returning:
`AttributeError: module 'sqlglot.expressions' has no attribute 'NULL'`

`t.schema().to_pandas()` works as expected, however, `(t.count().to_pandas(), len(t.schema()))` also breaks and returns the same error. Any thoughts?  I installed via poetry and it appears in my `.toml` as:
```
[tool.poetry.dependencies]
ibis-framework = {extras = ["duckdb"], version = "^7.2.0"}
```

I thought the error may be because the backend was incorrect. I tried checking the backend via `ibis.get_backend(t)` but that breaks. `ibis.set_backend("duckdb")` doesn't work.

My pandas version is `2.1.1` and duckdb is `0.9.1`.

**Comment by gforsyth** [comment, marked as answer]
Hey @JRocki!  There was an upstream breaking change in `sqlglot` that we didn't have a workaround for.  This is fixed on `main` and will be fixed in the next release coming next week.  In the interim, if you install `sqlglot < 20.11.0` that should avoid the breakage.  Sorry about the inconvenience!

**Reply by JRocki** [reply]
Ah ok sounds good, thanks for the quick answer!
